src/scitype.jl

```diff
@@ -72,78 +33,128 @@ In general, one cannot infer the scitype of an object of type
 
 Nevertheless, for some *restricted* machine types `U`, the statement
 `type(X) == AbstractArray{T, N}` for some `T<:U` already allows one
-deduce that `scitype(X) = AbstractArray{S,N}`, where `S` is determined
-by `U` alone. This is the case in the *MLJ* convention, for example,
-if `U = Integer`, in which case `S = Count`.
+deduce that `scitype(X, C) = AbstractArray{S, N}`, where `S` is determined
+by `U`, and convention `C` alone. This is the case in the `DefaultConvention` which is 
+used by *MLJ* , where for example, if `U = Integer`, then `S = Count`.
 
```

Comment:
please replace "which is used by *MLJ*" with "which is used in [ScientificTypes.jl](...)"